src/store/configureStore.js

```diff
@@ -1,11 +1,19 @@
 import { createStore, applyMiddleware, compose } from 'redux';
+import { persistStore, persistReducer } from 'redux-persist'
```

Comment:
Redux persist seems to be causing some errors?
![Screen Shot 2020-03-19 at 4 37 38 PM](https://user-images.githubusercontent.com/47183489/77108090-65e0d880-6a00-11ea-8950-af91e0168089.png)

Seems to happen every time I move a grid item